### ISSUE TITLE
Unable to import azurerm_monitor_scheduled_query_rules_alert

### ISSUE THREAD

**Issue by mochja** [Azure/aztfy#212]
Hello,

I was unable to import `azurerm_monitor_scheduled_query_rules_alert` resource using

```
aztfy version v0.6.0(18b614b)
```

```
aztfy resource -f '/subscriptions/D5CCAED7-D6BE-4ACF-A028-59369C7012CE/resourceGroups/my-rg/providers/microsoft.insights/scheduledqueryrules/Rule name with spaces'
Resource Type: azurerm_monitor_scheduled_query_rules_alert

...

Error: failed to import /subscriptions/D5CCAED7-D6BE-4ACF-A028-59369C7012CE/resourceGroups/my-rg/providers/Microsoft.Insights/scheduledqueryrules/Rule name with spaces as azurerm_monitor_scheduled_query_rules_alert.res-0: exit status 1
Error: parsing Resource ID "/subscriptions/D5CCAED7-D6BE-4ACF-A028-59369C7012CE/resourceGroups/my-rg/providers/Microsoft.Insights/scheduledqueryrules/Rule name with spaces": ID was missing the `scheduledQueryRules` element
```

I found this fix: https://github.com/hashicorp/terraform-provider-azurerm/pull/17932/files

But I am not able to convince the azcli to use `scheduledQueryRules` instead of `scheduledqueryrules`

**Comment by magodo** [comment]
@mochja Thank you for submitting this! The issue is already detected and fixed at: https://github.com/magodo/aztft/commit/aee403992334ac2d171a371fdf093eeea794cb4c#diff-612a386b491da3c67f6b96706d95ff5aea2bb99d91b01bf68c922444c967fccdL6890-R7008, which is already included in the current main branch. I'll make a new release soon so that users don't need to build from the main branch.

**Comment by magodo** [comment]
This has been fixed in v0.7.0.
